source/drivers/null/ur_null.cpp

```diff
@@ -9,89 +9,121 @@
  */
 #include "ur_null.h"
 
-namespace driver
-{
-    //////////////////////////////////////////////////////////////////////////
-    context_t d_context;
+namespace driver {
```

Comment:
Why are there changes in this file?

Reply:
It wasn't spec conformant. I had to fix some error codes that were being returned and added some additional cases. 

Not sure about how to deal with the formatting changes - Since this is not an autogenerated file we don't have a formatting style I guess.

Reply:
Oh, I assumed it was autogenerated.

As for formatting, yeah we don't have it yet but we have #101 to track it.